registration-processor-default.properties

```diff
@@ -473,8 +474,12 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1
 ## Printing
 mosip.registration.processor.credentialtype=euin
 mosip.registration.processor.encrypt=false
-mosip.registration.processor.issuer=mpartner-default-print 
 mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
+mosip.registration.processor.digitalcard.credentialtype=PDFCard
+mosip.registration.processor.print.credentialtype=euin
+mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
```

Comment:
Add detailed comment with example on the property value's format.

Reply:
taken care